### ISSUE TITLE
feature request: libjq documentation

### ISSUE THREAD

**Issue by sergey-safarov** [jqlang/jq#2829]
On the Fedora dist I can see `jq` package has `libjq` library.
```sh
[safarov@safarov-dell tamu-lab]$ rpm -ql jq | grep libjq
/usr/lib64/libjq.so.1
/usr/lib64/libjq.so.1.0.4
```
I want use jq functionality from other program natively, without calling bash and serializing/deserializing json.
I can see preent `.h` files
```sh
[safarov@safarov-dell tmp]$ rpm -ql jq-devel
/usr/include/jq.h
/usr/include/jv.h
/usr/lib64/libjq.so
```
But no documentation how to use functions in the library.
Could you extend documentation of library calls?

**Comment by emanuele6** [comment]
Yes, it would be nice to have manual pages for `jv_*()`, and other libjq public functions. ^^

**Comment by nicowilliams** [comment]
Yes, I think we can start thinking about the stability of the `jv` and `jq` APIs now.

**Comment by dad4x** [comment]
Would it be welcome for someone to do man pages for them?   I'm in need of something like libjq for a project, and may have to do something for my own understanding.

**Comment by dad4x** [comment]
jq_set_colors() seems underbaked as an API.    Does it store the color mapping into a global?   Probably it should be bound to a jq_state with a get and set, and maybe it should be a jv instead of a const char*?

(I am looking to use this in a server, where different concurrent operations might have different color maps).
